config/initializers/scrna_config.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# This hash is used to store constants used in pooling and chip loading calculations for samples in the scRNA pipeline.
+Rails.application.config.scrna_config = {
```

Comment:
Should these (constants) all be in capital letters like class constants?

Reply:
I don't really know! I'm happy as is.